### ISSUE TITLE
Integrate maven tooltip generator plugin into build workflow

### ISSUE THREAD

**Issue by gashcrumb** [syndesisio/syndesis-documentation#23]
Follow on from https://github.com/syndesisio/syndesis-documentation/issues/20 the plugin is there but not part of the build.

For a first draft we should just ensure the generator is run as part of building the documentation project and generate the tooltip JSON into `target`, at the very least then we'll have a json file that could be copied into the UI project somewhere.

@rhuss FYI

related to https://github.com/syndesisio/syndesis-project/issues/47

**Comment by rhuss** [comment]
I would go even so far that this file is created somewhere directly below a fixed directory in this repo, and then committed to git). Then, the syndesis-ui build could directly download this file from the raw URL from github when it fetches the tooltips.

This should be really a big issue, however would need to know a bit more about the dev process in this repo (e.g. how and when the documentatio is rebuild)

**Comment by gashcrumb** [comment]
Yeah, that'd probably better I guess, as otherwise we'd have to know that changes occurred.  I guess we'd want to know about those anyways so the right controls can be annotated with tooltips.  Whatever's easiest I guess :-)
